build.gradle

```diff
@@ -1,10 +1,6 @@
-allprojects {
-    def deployVersion = properties['DeployVersion']
-    if (deployVersion != null) version = deployVersion
-}
-
 buildscript {
-    if (rootProject.properties['kotlinSnapshot'] != null) {
+    ext.useKotlinSnapshot = rootProject.properties['kotlinSnapshot'] != null
+    if (useKotlinSnapshot) {
```

Comment:
Is this block still necessary after adding `allprojects` with the same check?

Reply:
root `buildscript` executes earlier than anything. Setting `kotlin_version` in `allprojects` is required because each project contains its own ext variable, thanks to `gradle.properties` file.